src/sentry_plugins/asana/plugin.py

```diff
@@ -155,7 +130,7 @@ def create_issue(self, request, group, form_data, **kwargs):
                 workspace=self.get_option('workspace', group.project), data=form_data
             )
         except Exception as e:
-            self.raise_error(e)
+            self.raise_error(e, identity=client.auth)
```

Comment:
this is the main goal here, so we can attach the identity to 'InvalidIdentity' errors

Reply:
thanks, helpful